.travis.yml

```diff
@@ -14,16 +14,26 @@ dist: trusty
 compiler: clang # per: https://docs.travis-ci.com/user/languages/c
 # https://docs.travis-ci.com/user/environment-variables
 env:
- - CXX=clang++ DEBUG='nightmare*'
- # - CXX=clang++ DISPLAY=':99.0' DEBUG='nightmare*'
-# according to travis docs, the services -xvfb line will set DISPLAY
 branches:
   only:
     - master # https://github.com/okTurtles/group-income-simple/issues/58
 addons:
   apt:
     packages:
       - xvfb
+cache:
```

Comment:
Check if `xvfb` is still required. If it's not then you can remove this part.

Reply:
I believe not, so I removed it all.